Mathlib/Algebra/Ring/UnitEquivs.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
```

Comment:
Is this a copy-paste error?
```suggestion
Copyright (c) 2023 Eric Rodriguez. All rights reserved.
```
Same remark for `Authors` field on line 4, though maybe this is co-authored?